### DISCUSSION TITLE
headerHtml doesn't work for me

### DISCUSSION THREAD

**Discussion by ahmetkocabiyik** [spatie/browsershot#837] [Q&A]
I tried lots of time but headerHtml not produce anyting. It only show normal content view. showBrowserHeaderAndFooter() method working well but headerHtml or footerHtml doesn't product anything. I use browsershot 4.0.2 version and puppeteer 22.6.1 version in laravel 11. I don't understand what i am doing wrong

```php
Browsershot::html(view("pdf.estimate",compact("estimate")))
            ->showBackground()
            ->headerHtml('<p>test</p>')
            ->margins(10,10,10,10)
            ->format("a4")
            ->save("test5.pdf");
```

**Comment by cezaro** [comment]
I know it's been few months, but now I'm facing same problem and found solution https://stackoverflow.com/a/58867835
